sklearn/feature_extraction/stop_words.py

```diff
@@ -43,3 +43,341 @@
     "who", "whoever", "whole", "whom", "whose", "why", "will", "with",
     "within", "without", "would", "yet", "you", "your", "yours", "yourself",
     "yourselves"])
+
+TURKİSH_STOP_WORDS = frozenset([
```

Comment:
There are an error unicode ın Turkısh language. In turkish language, "ı" equal to "I" and "i" equal to "İ".